convert_samples.ps1

```diff
@@ -0,0 +1,74 @@
+# PowerShell script to convert all 9 dataset samples: Raw -> Standardized -> SFT (OpenHands format)
```

Comment:
I think generally we work on mac with bash, so it'd be better to have a bash script if any. You can also use it on windows in WSL